### PR TITLE
Replace all NAMESPACE refs with CDI_NAMESPACE

### PR DESCRIPTION
**Pull request by arnongilboa**
Signed-off-by: Arnon Gilboa <agilboa@redhat.com>

**What this PR does / why we need it**:
Makes build more readable and maintainable, as we now have a single NS variable.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: manifests/templates/imageio.yaml.in [hunk 1]
```diff
@@ -45,7 +45,7 @@ spec:
         ports:
         - containerPort: 12346
         env:
-          - name: NAMESPACE
+          - name: CDI_NAMESPACE
```
Comment (brybacki) [inline manifests/templates/imageio.yaml.in:48]: Are you sure?

Comment (arnongilboa) [reply manifests/templates/imageio.yaml.in:48]: Not at all. Fat fingers...

**Pull request merged**